pkg/system/network/linux.go

```diff
@@ -159,6 +159,18 @@ func (t NetworkManagerLinux) SetPendingNetwork(selectedNetwork dogeboxd.Selected
 	return t.sm.SetNetwork(ns)
 }
 
+func (t NetworkManagerLinux) TestConnect(nixPatch dogeboxd.NixPatch) error {
```

Comment:
This doesn't seem like it needs a nix patch arg

Reply:
I've removed the argument